docs/process/release-planning.md

```diff
@@ -1,7 +1,77 @@
 # Release Planning
 
-We plan work at two levels of granularity: first at the marketing release level (1.1, 1.2, etc.) and then at the milestone level. Marketing releases are made up of multiple milestones. Milestones should contain roughly a month’s worth of work.
+This document outlines our process for planning and scheduling releases, so you
+can familiarize yourself with the flow of work from opening an issue to seeing
+it published in a release.
 
-We’ll release updates as needed or when a milestone is completed.
+## Releases
 
-Large features and user-visible changes are feature-flagged, to be unflagged in the next marketing release. Beta and dev builds will have all feature flags enabled.
+We organize releases in two ways - marketing and milestones:
+
+ - Marketing releases are what we use to represent planned features, and are
+   high-level goals
+      - **for example: 1.4, 1.5, etc.**
+ - Milestones are used to track issues and pull requests associated with an
+   upcoming release, and can be [followed on GitHub](https://github.com/desktop/desktop/milestones)
+      - **for example: 1.4.1, 1.4.2, 1.5.0, etc.**
+
+We aim to ship updates to production approximately every two weeks, to ensure a
+continuous flow of improvements to our users. Track our progress in the
+[current milestones](https://github.com/desktop/desktop/milestones?direction=desc&sort=completeness&state=open).
+
+## Scheduling Pull Requests
+
+Pull Requests for user-facing changes should have a milestone associated with
+them, to indicate when they should be merged.
+
+### Features
+
+Pull Requests associated with features for our marketing releases should have a
+milestone defined as soon as possible, to indicate the anticipated release and
+help track.
+
+Pull requests for new features should leverage feature flags, so we can control
```

Comment:
should we include a link to https://github.com/desktop/desktop/blob/master/docs/technical/feature-flagging.md here?

Reply:
6d507ff